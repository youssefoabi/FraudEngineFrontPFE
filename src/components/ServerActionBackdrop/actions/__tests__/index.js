import { itShouldTestActionCreators } from '@cdiscount/jest-utils';

import * as actions from '../';

describe('Actions', () => {
  itShouldTestActionCreators(actions);
});
