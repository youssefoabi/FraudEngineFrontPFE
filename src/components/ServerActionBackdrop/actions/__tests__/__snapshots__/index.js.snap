// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scopes actions PENDING_REQUEST 1`] = `"PENDING_REQUEST"`;

exports[`Scopes actions RESET_PENDING_REQUEST 1`] = `"RESET_PENDING_REQUEST"`;
